lib/models.js

```diff
@@ -62,6 +62,10 @@ class DataElementSpecifications {
     return this.all.filter(de => de.isEntry);
   }
 
+  get abstracts() {
+    return this.all.filter(de => de.isAbstract);
+  }
```

Comment:
I spent some time thinking about it and I couldn't think of a use case for myself. Potentially, there might be a usage in the online viewer, if for some reason we wanted to display all of the abstract elements in isolation? (Although that might be handled elsewhere in the toolchain, and not here).